addons/mail/static/src/discuss/call/common/chat_window_patch.scss

```diff
@@ -1,4 +1,4 @@
-.o-mail-ChatWindow-command {
+.o-mail-ChatWindow-command.o-quick {
     @media (hover: none) {
         .fa-phone, .fa-video-camera {
             color: $success !important;
```

Comment:
Actually only the outline effect is relevant, with the change to use static color even in desktop.

(This is intended to give better visibility in discuss sidebar actions, and for consistency this is generalized everywhere including discuss header and chat window quick actions)